src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceEducation.vue

```diff
@@ -39,7 +39,13 @@
               v-model="country"
               item-title="countryName"
               item-value="countryId"
-              :rules="[Rules.required('Select your country', 'countryId')]"
+              :rules="[
+                Rules.required('Select your country', 'countryId'),
+                (value) =>
+                  !isDraftApplicationAssistantRenewal ||
+                  value?.countryId === configStore.canada?.countryId ||
+                  'New course must be part of a recognized program',
+              ]"
```

Comment:
We have a conditionalWrapper that we can leverage for this 

https://github.com/bcgov/ECC-ECER/blob/765d191c66985ba2eded852b73644e751299afa9/src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/utils/formRules.ts#L227

it's being used in this component as well so you can copy that pattern. 

You just slot in a condition. 

When it's true, it will enforce the rule. Otherwise it will ignore. 

I think it is more straightforward to read since it separates our condition 

and then our rule.

Reply:
here's an example of it being used 

![image](https://github.com/user-attachments/assets/76166ed8-4149-4690-ba67-33ba8411d751)